platform/broadcom/sai.dep

```diff
@@ -3,12 +3,16 @@
 SPATH       := $($(BRCM_SAI)_SRC_PATH)
 DEP_FILES   := $(SONIC_COMMON_FILES_LIST) platform/broadcom/sai.mk platform/broadcom/sai.dep   
 DEP_FILES   += $(SONIC_COMMON_BASE_FILES_LIST)
+# Get the Latest HTTP Header and calculate the SHA value as it is a softlink that always points to LATEST_INT_OCP_SAI_X.X.X
```

Comment:
>HTTP Header [](start = 17, length = 11)

Is it better to download the whole file and calc the SHA value? It is tens of MB so not a big issues.

Reply:
Adding the complete file content for the SHA calculation might cause a delay in the initial build phase especifically in the slow network as these files are included in the begining of the make process.  The HTTP header itself is sufficient and it is always reflects the change if there is any modification in the content.  I feel this is sufficient, your thoughts ?